rotkehlchen/tasks/manager.py

```diff
@@ -891,6 +893,24 @@ def _maybe_delete_past_calendar_events(self) -> Optional[list[gevent.Greenlet]]:
             database=self.database,
         )]
 
+    def _maybe_query_graph_delegated_tokens(self) -> Optional[list[gevent.Greenlet]]:
+        """
+        Search for DelegationTransferredToL2
+        """
+        if should_run_periodic_task(self.database, DBCacheStatic.LAST_GRAPH_DELEGAIONS_CHECK_TS, DAY_IN_SECONDS) is False:  # noqa: E501
+            return None
+
+        ethereum_manager: EthereumManager = self.chains_aggregator.ethereum
+        eth_txns: EthereumTransactions = ethereum_manager.transactions  # type: ignore
```

Comment:
should the task also run when we decode a transaction? Maybe clear the last check ts